app/models/solidus_bolt/bolt_configuration.rb

```diff
@@ -37,10 +37,9 @@ def environment_url
     end
 
     def embed_js
-      if BoltConfiguration.environments[environment].zero?
```

Comment:
Can we just use the safe navigation operator? Like `if BoltConfiguration.environments[environment]&.zero?`

Reply:
Nope, because in that way it will set the environment to production for default when the env is not set.

Reply:
Since we have now added a migration, this change is not required